### PR TITLE
Update typography

### PR DESCRIPTION
**Pull request by AtkishkinVlad**
Add next/font

### CODE REVIEW

File: components/ManualPage/ManualPage.module.css [hunk 1]
```diff
@@ -16,64 +20,51 @@
...
-.templateImage {
-    width: 100%;
-    position: relative;
-    border-radius: 4px;
+.Manual__image {
+    border-radius: 16px;
     box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+    object-fit: contain;
+    width: 100% !important;
+    position: relative !important;
```
Comment (a-chabin) [inline components/ManualPage/ManualPage.module.css:31]: Let's remove important. Why are they needed?

File: components/ManualPage/ManualPage.module.css [hunk 2]
```diff
@@ -16,64 +20,51 @@
...
+.Manual__image__container {
+    position: relative;
+    width: 100%;
+    text-align: center;
+    margin-top: 1rem;
 }
 
-.heading2 {
-    font-size: 2.5rem;
+.Manual__image__container > div {
```
Comment (a-chabin) [inline components/ManualPage/ManualPage.module.css:42]: Bad specifity: 0,1,1.

File: components/NotionTypes/Bookmark/link.module.css [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
 }
 
 .link:hover {
-    color: rgb(89, 89, 89) !important;
+    color: #E63223 !important;
+}
+
+@media screen and (prefers-color-scheme: dark) {
+    .link {
+        color: E63223 !important;
```
Comment (a-chabin) [inline components/NotionTypes/Bookmark/link.module.css:17]: missed `#` 😨

File: pages/_app.jsx [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+        },
+        {
+            path: '../styles/fonts/IsetSans-Regular.woff2',
+            weight: '400',
+        },
+        {
+            path: '../styles/fonts/IsetSans-Regular-Italic.woff2',
+            weight: '400',
+            style: 'italic',
+        },
```
Comment (a-chabin) [inline pages/_app.jsx:29]: Let's move this to separate file

File: next.config.js [hunk 5]
```diff
@@ -1,6 +1,17 @@
-module.exports = {
-    api: {
-        HOST: process.env.NEXT_PUBLIC_HOST,
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+    env: {
+        HOST: 'http://guides-api-test.ekaterinburg.design:48700',
```
Comment (a-chabin) [inline next.config.js:6]: Let's move it in a `.env` file

File: components/ManualPage/ManualPage.module.css (hunk 1 above)
Comment (AtkishkinVlad) [reply components/ManualPage/ManualPage.module.css:31]: I looked at this [article](https://dev.to/felixhaeberle/responsive-fix-for-the-next-js-image-component-1351) to get adaptive pictures with `next/image` (without important it doesn't work)

File: components/ManualPage/ManualPage.module.css [hunk 6]
```diff
@@ -69,7 +69,7 @@
         padding-left: 60px;
         max-width: 1400px;
         margin-left: max(20%, 376px);
-        overflow: hidden;
+        overflow: visible;
```
Comment (a-chabin) [inline components/ManualPage/ManualPage.module.css:72]: Let's remove the default this rule with default `overflow`  value

File: components/CommonLinks/CommonLinks.module.css [hunk 7]
```diff
@@ -1,7 +1,49 @@
 .CommonLinks__container {
-    padding: 15px;
+    position: fixed;
+    display: flex;
+    gap: 12px;
```
Comment (a-chabin) [inline components/CommonLinks/CommonLinks.module.css:4]: Let's use either margin for children, or gap for the parent

File: components/TableOfContents/TableOfContents.module.css [hunk 8]
```diff
@@ -212,6 +212,7 @@
 
     .innerlinkContainerList {
         background-color: #404040;
+
```
Comment (a-chabin) [inline components/TableOfContents/TableOfContents.module.css:215]: empty line 🧐

File: components/NotionTypes/Bookmark/link.module.css [hunk 9]
```diff
@@ -1,13 +1,19 @@
 .link {
```
Comment (a-chabin) [inline components/NotionTypes/Bookmark/link.module.css:1]: link.module.css → Link.module.css

**Comment by a-chabin** [comment]
Lets rename `link.module.css` → `Link.module.css` for consistency

File: components/CommonLinks/CommonLinks.module.css [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+    text-decoration: none;
+    font-weight: 400;
+    box-shadow: none;
+    color: black;
+    vertical-align: middle;
+    text-decoration: none;
+    border-radius: 10px;
+    transition: 0.2s;
+    padding: 15px;
+    font-size: 1rem;
```
Comment (a-chabin) [inline components/CommonLinks/CommonLinks.module.css:26]: Let's use `px` and `rem`. Our application has become much more component-based to depend on global values

Comment (a-chabin) [reply components/CommonLinks/CommonLinks.module.css:26]: We can put it in a separate Issue

File: components/CommonLinks/CommonLinks.jsx [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+                width={60}
+                height={60}
+                className={styles.CommonLinks__icon}
+            />
+            <Image
+                src="/ecosystem.svg"
+                width={60}
+                height={60}
+                className={styles.CommonLinks__icon}
+            />
```
Comment (a-chabin) [inline components/CommonLinks/CommonLinks.jsx:21]: Are the images not clickable?

File: components/NotionTypes/Table/Table.module.css [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+    margin-bottom: 20px;
+    max-width: 100%;
+    overflow-x: scroll;
+}
+
+.Table {
+    width: 100%;
+    border: none;
+    border-collapse: separate;
+    border-spacing: 0;
```
Comment (a-chabin) [inline components/NotionTypes/Table/Table.module.css:13]: Let's use

```
border-collapse: collapse;
```

instead
```
border-collapse: separate;
border-spacing: 0;
```

File: components/CommonLinks/CommonLinks.module.css (hunk 7 above)
Comment (AtkishkinVlad) [reply components/CommonLinks/CommonLinks.module.css]: Indents were not needed at all here 🙂 

P.S I will remember about the choice between gap and margin

**Pull request closed without merging**